javascript/async-functions/js-async-functions_sleep/js/index.js

```diff
@@ -8,18 +29,21 @@ const body = document.body;
 
 button3.addEventListener("click", async () => {
   icon.textContent = "💤";
-  sleep(3);
+  await sleep(3);
   icon.textContent = "⏰";
 });
 
-button5.addEventListener("click", () => {
+button5.addEventListener("click", async () => {
   icon.textContent = "💤";
+  await sleep(5);
   icon.textContent = "⏰";
 });
 
 function handleSleepFor10Button() {
```

Comment:
I think this function must also be async, right? Like `async function`. Or does it work like this?